data/mods/blazemod/items/tools/blob_tools.json

```diff
@@ -0,0 +1,651 @@
+[
```

Comment:
Another separate file formed, this one by pulling all of the `"type": "TOOL"` out of blaze_blob.json